hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/om-overview.html

```diff
@@ -49,6 +45,35 @@ <h2>Status</h2>
     </tbody>
 </table>
 
+<div ng-show="$ctrl.overview.jmx.RatisRoles.length > 0">
+    <h2>OM Roles (HA)</h2>
+    <table class="table table-striped table-bordered" class="col-md-6">
+        <thead>
+        <tr>
+            <th>Host Name</th>
+            <th>Node ID</th>
+            <th>Ratis Port</th>
+            <th>Role</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr class="om-roles-background" ng-repeat="roles in $ctrl.overview.jmx.RatisRoles"
+            ng-if="$ctrl.role.Id == roles[2].value">
+            <td>{{roles[3].value}}</td>
```

Comment:
there is assumption that index as in order represent hostname, nodeid, port, role. Do this is valid for all scenario? or need get based on string in json map ?